### ISSUE TITLE
[request] Add metadata to Documents and Tables

### ISSUE THREAD

**Issue by asitemade4u** [gristlabs/grist-core#98]
Something I find missing in Grist is the ability to add **metadata** to Documents, Tables and even Columns.
For example:

- description, use cases, explanation, instructions: text file
- main author with email

This feature is available in Airtable and comes very handy.

**Comment by paulfitz** [comment]
Sounds like a good idea. Would you be interested in the ability to set up document tours, like the templates at https://templates.getgrist.com/ have?

**Comment by asitemade4u** [comment]
Yes that would also be a great idea, but seems more complex to implement than plain text fields attached to main Grist objects.

**Comment by rambip** [comment]
> Sounds like a good idea. Would you be interested in the ability to set up document tours, like the templates at https://templates.getgrist.com/ have?

Wow, that's an invaluable tool ! I really like the "Grist" spirit of this feature.

**Comment by paulfitz** [comment]
@rambip document tours did get written up: https://support.getgrist.com/document-tours/

Some of @asitemade4u's suggestions have since been implemented, but by no means all - https://support.getgrist.com/newsletters/2023-05/#column-and-widget-descriptions

**Comment by rambip** [comment]
> [@rambip](https://github.com/rambip) document tours did get written up: https://support.getgrist.com/document-tours/
> 
> Some of [@asitemade4u](https://github.com/asitemade4u)'s suggestions have since been implemented, but by no means all - https://support.getgrist.com/newsletters/2023-05/#column-and-widget-descriptions

Quick question: is it possible to use markdown in the Tour step, or at least to span multiple lines ? Otherwise it's quite hard to read. I can open an issue if needed.

**Comment by dsagal** [comment]
> Quick question: is it possible to use markdown in the Tour step, or at least to span multiple lines ? Otherwise it's quite hard to read. I can open an issue if needed.

I believe the answer is no. But now that cells support markdown, it seems sensible and straightforward to support. You are welcome to open an issue!
